Python/pywarpx/picmi.py

```diff
@@ -355,6 +355,13 @@ def init(self, kw):
         self.max_grid_size = kw.pop('warpx_max_grid_size', 32)
         self.blocking_factor = kw.pop('warpx_blocking_factor', None)
 
+        self.potential_xmin = 0
```

Comment:
It would be better if this was `self.potential_xmin = None` so that potential_xmin does not appear in the input data, and the same for xmax. Similarly, the default values of the others should be None, i.e. `self.potential_ymin = kw.pop('warpx_potential_zmin', None)`. When the value is None, the input parameter is ignored and the default value in the compiled code is used.